docs/getting-started/tutorial-sample-discover.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[tutorial-sample-discover]]
+=== Using Discover
+
+Another way to interactively explore your data is in the Discover application.   
+Here the data is presented as a table of documents.  As in the Dashboard application, 
+you can submit search queries and filter the search results. 
+
+. In the side navigation, select *Discover*.
++
+The current index pattern appears below the filter bar. An
+index patterns tell {kib} which {es} indices you want to explore.
+You’ll learn more about index patterns in the next tutorial, 
```

Comment:
"The index pattern..." I'm not sure I'd bank on them going through both tutorials...would probably x-ref "more info about index patterns" instead.